src/components/TextEditor/TextEditor.vue

```diff
@@ -346,4 +354,76 @@ img.ProseMirror-selectednode {
 .tag-suggestion-active {
   background-color: var(--surface-gray-2, #f3f3f3);
 }
+
+.scrollable-tables table {
```

Comment:
All table related CSS should be in a separate CSS file imported in TextEditor.vue